geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ServerConnection.java

```diff
@@ -764,6 +772,16 @@ protected void doHandshake() {
     }
   }
 
+  private boolean incrementConnectionsProcessing() {
```

Comment:
why not move this method to the ServerConnectionCollection class since it only works with it?